### PR TITLE
Add aprende python

### PR DESCRIPTION
**Pull request by quobit**
## What does this PR do?
Add resource

### Description
Active and updated book to learn python in spanish made by a CS teacher

### Why is this valuable (or not)?
It can be used to teach from secondary/high school to upper levels.

### How do we know it's really free?
The license is in the front page (GPLv3).

### For book lists, is it a book? For course lists, is it a course? etc.
Yes, it is a book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-es.md [hunk 1]
```diff
@@ -327,6 +327,7 @@
 ### Python
 
 * [Aprenda a pensar como un programador (con Python)](https://argentinaenpython.com/quiero-aprender-python/aprenda-a-pensar-como-un-programador-con-python.pdf) (PDF)
+* [Aprende Python](https://aprendepython.es) (HTML)
```
Comment (davorpa) [inline books/free-programming-books-es.md:330]: Add author attributions to fit with licenses

```suggestion
* [Aprende Python](https://aprendepython.es) - Sergio Delgado Quintero (HTML)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to apply some changes. When done, request for a review again using GitHub :arrows_clockwise: button. Meanwhile, draft status can be used to work on it if you want.

**Review by CleanMachine1 — APPROVED** [review]
Once davorpa's suggestion is added then LGTM

File: books/free-programming-books-es.md (hunk 1 above)
Comment (quobit) [reply books/free-programming-books-es.md]: Done! Thank you!

**Review by davorpa — APPROVED** [review]

File: books/free-programming-books-es.md [hunk 2]
```diff
@@ -327,6 +327,7 @@
 ### Python
 
 * [Aprenda a pensar como un programador (con Python)](https://argentinaenpython.com/quiero-aprender-python/aprenda-a-pensar-como-un-programador-con-python.pdf) (PDF)
+* [Aprende Python](https://aprendepython.es) - Sergio Delgado Quintero (HTML)
```
Comment (davorpa) [inline books/free-programming-books-es.md:330]:
```suggestion
* [Aprende Python](https://aprendepython.es) - Sergio Delgado Quintero (HTML, PDF)
```

**Comment by eshellman** [comment]
Thanks @quobit !

**Pull request merged**